packages/@okta/vuepress-site/docs/release-notes/2023/index.md

```diff
@@ -4,6 +4,20 @@ title: Okta API Products release notes 2023
 
 ## March
 
+### Weekly release 2023.03.2
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Bugs fixed in 2023.03.2](#bugs-fixed-in-2023-03-2) | March 22, 2023 |
+
+#### Bugs fixed in 2023.03.2
+
+* In some cases, groups with a `status` of INACTIVE were synchronized with the reporting database as ACTIVE. (OKTA-589084)
+
+* Requests to the Policies API (`PUT /policies/${defaultIdpPolicy}/rules/${IdpRule}`) with an empty `userIdentifier` parameter returned an HTTP 500 Internal Server error. (OKTA-565856)
```

Comment:
did you see the edit spreadsheet about this one? They removed the HTTP 500 Internal Server portion of the blurb.